README.md

```diff
@@ -77,6 +77,15 @@ scoop install act
 yay -S act
 ```
 
+### [COPR](https://copr.fedorainfracloud.org/coprs/rubemlrm/act-cli/) (Linux)
+
+[![copr-shield](https://img.shields.io/copr/version/act)](https://copr.fedorainfracloud.org/coprs/rubemlrm/act-cli/)
```

Comment:
Doesn't seem like there is shield for COPR

Reply:
My bad. I will do a commit to fix this. Thanks a lot for the reply. Can I just leave a link like was made for the unix step?

Reply:
Sure

Reply:
done thanks a lot